client/src/pages/cejst.tsx

```diff
@@ -15,45 +15,51 @@ const CEJSTPage = ({location}: IMapPageProps) => {
   //   We will bring back later when we have interactive controls.
   return (
     <Layout location={location}>
-      <main id="main-content" role="main">
```

Comment:
We should still have a main, right? This is one of the warnings we've been getting from Lighthouse - "Ensures all page content is contained by landmarks". I was able to fix this locally for a different page by adopting the guidance here: https://www.w3.org/WAI/tutorials/page-structure/regions/

Reply:
I want to say this might be ok to delete, since the layout component already has a `main`. I think this `main` under question, is the 2nd nested `main` that is being flagged by the a11y tool that Glennette is using

Reply:
The issue is that you shouldn't have more than one `<main>` per page (I think the accessibility report touched on this). 

The `layout.tsx` has this element, so it shouldn't be in any per-page templates.

Reply:
ah ok this makes sense